services/datagrid-service-template.yaml

```diff
@@ -214,6 +214,10 @@ parameters:
   displayName: Total Memory
   name: TOTAL_CONTAINER_MEM
   value: "512"
+- description: Sets the total storage capacity, in GiB, available to the application.
```

Comment:
Wondering if we should specify that this is persistent storage? Users might wonder what's the difference between container memory and storage capacity?